tests/lib/snaps/test-snapd-remodel-pc-18/grub.cfg

```diff
@@ -0,0 +1,42 @@
+set default=0
+set timeout=3
+
+insmod part_gpt
+insmod ext2
+
+if [ -s $prefix/grubenv ]; then
+  load_env
```

Comment:
nitpick: the indentation in other parts of this file is 4 spaces

Reply:
This is a verbatim copy of what's inside the pc snap in 18 channel